ramalama/cli.py

```diff
@@ -442,15 +448,15 @@ def run_container(args):
     if args.nocontainer or in_container() or sys.platform == "darwin":
         return False
 
-    name = _name()
-    if hasattr(args, "name") and args.name is not None:
-        name = args.name
-
     conman = container_manager()
     if conman == "":
         return False
 
-    home = os.path.expanduser("~")
+    if hasattr(args, "name"):
+        name = args.name
+    else:
```

Comment:
I'm starting to learn python doesn't have `if` scope, variables declared inside the if scope can be used outside the scope... I must use if/else more in python to avoid setting things twice...